### ISSUE TITLE
Error: 'createDHARMa' is not an exported object from 'namespace:DHARMa'

### ISSUE THREAD

**Issue by gavanmcgrath** [florianhartig/BayesianTools#59]
Hi Florian,
 Just had a go at the update. Running the package ?VSEM example (not my previous expansions), produced the above error at plotTimeSeries . A plot is produced so the error comes after the initial plot and data are created. Looks like a function is expected from the DHARMa package. I get the same error running my own model.
Cheers,
G

**Comment by florianhartig** [comment]
this is weird - can you check if you have the DHARMa package installed? This is needed because we added a call to the DHARMa package in the development version, but running 

`devtools::install_github(repo = "florianhartig/BayesianTools", subdir = "BayesianTools", dependencies = T, build_vignettes = T)`

should have installed DHARMa - did you run the install with dependencies = T?

**Comment by gavanmcgrath** [comment]
Thanks again,

So no I'd not previously installed with dependencies = TRUE. I had
been manually installing packages when packages were reported missing.
However, I was not getting this missing package related message when I used
the cran package, so had DHARMa been installed then?

Subsequent to your email I did set dependencies = TRUE and followed your
install instructions though I eventually had success by doings
an install_github DHARMa and then a install_github  BayesianTools with
dependencies = TRUE and force = TRUE for both.
Now it works.

Your help has been much appreciated.

Gavan
